src/energyplus/ForwardTranslator.cpp

```diff
@@ -108,9 +108,13 @@ namespace energyplus {
     m_excludeSQliteOutputReport = false;
     m_excludeHTMLOutputReport = false;
     m_excludeVariableDictionary = false;
+    m_excludeSpaceTranslation = false;
   }
 
   Workspace ForwardTranslator::translateModel(const Model& model, ProgressBar* progressBar) {
+
+    // When m_excludeSpaceTranslation is false, could we skip the (expensive) clone since we aren't combining spaces?
+    // No, we are still doing stuff like removing orphan loads, spaces not part of a thermal zone, etc
```

Comment:
Agree. I can't readily provide an example, but im confident that the translated model (a copy) is mutated in other ways besides combine spaces in various parts of the translation process. The unwritten rules of the translator are that it is ok to mutate the model.

Reply:
A couple lines below in translateModelPrivate  it starts by removing orphan loads etc.
While we could do things differently (catch that in the FT function specific to each class), this is only going to increase complexity of this already complex PR.
I might revisit this for performance reasons in the future.